packages/cli/src/commands/migrate/tasks/config-lint.ts

```diff
@@ -10,8 +13,66 @@ export async function lintConfigTask(): Promise<ListrTask> {
         task.output = `Create .eslintrc.js from config`;
         await _ctx.userConfig.lintSetup();
       } else {
-        task.skip(`Skip creating .eslintrc.js since no custom config is provided`);
+        const configPath = resolve(options.basePath, '.eslintrc.js');
+        if (existsSync(configPath)) {
+          task.output = `${configPath} already exists, adding typescript-related config`;
+          task.title = `Update eslintrc.js`;
+          await updateLintConfig(configPath);
+        } else {
+          createLintConfig(configPath);
+        }
       }
     },
   };
 }
+
+async function updateLintConfig(path: string): Promise<void> {
+  /* eslint-disable-next-line @typescript-eslint/no-var-requires */
+  const oldConfig = require(path);
+  const newConfig = {
+    ...oldConfig,
+    plugins: oldConfig.plugins
+      ? [...oldConfig.plugins, '@typescript-eslint']
+      : ['@typescript-eslint'],
+    extends: oldConfig.extends
+      ? [
+          ...oldConfig.extends,
+          'plugin:@typescript-eslint/recommended',
+          'plugin:@typescript-eslint/eslint-recommended',
+        ]
+      : ['plugin:@typescript-eslint/recommended', 'plugin:@typescript-eslint/eslint-recommended'],
+  };
+  const configStr = 'module.exports = ' + JSON.stringify(newConfig, null, 2);
+  outputFileSync(path, configStr);
```

Comment:
I don't think we should clobber over the applications `.eslintrc.js`. It's likely better to create a different file path and we let the user know we created the file and are using it for linting and formatting.

Reply:
A few ways we can handle this:

1. create a new file for rehearsal-eslint in the root of the project with our curated set of plugins. add an extends in the root .eslintrc with a path to this newly created file eg.
```
  extends: [
    'plugin:@typescript-eslint/eslint-recommended',
    'plugin:@typescript-eslint/recommended',
    './.rehearsal-eslintrc.js',
  ],
```

2. read the root .eslintrc and clone it into the new rehearsal-eslint file, leaving the root .eslintrc alone. now the rehearsal-eslint

3. create a new eslint plugin eg. rehearsal and add the dependency and add the extend to the root .eslintrc

Reply:
Done